lib/views/general/_popup_banner.html.erb

```diff
@@ -1 +1 @@
-<%#- anything you put here will appear as a popup warning on every screen in Alaveteli -%>
+<%= link_to 'Olağanüstü Hal'de Bilgi Edinme', 'http://platform24.org/Content/Uploads/Editor/OHALDE%20BILGI%20EDINME%20(1).htm' -%>
```

Comment:
Needs to be in double quotes otherwise the `Hal'de` mistakenly closes the string

```diff
-<%= link_to 'Olağanüstü Hal'de Bilgi Edinme', 'http://platform24.org/Content/Uploads/Editor/OHALDE%20BILGI%20EDINME%20(1).htm' -%>
+<%= link_to "Olağanüstü Hal'de Bilgi Edinme", 'http://platform24.org/Content/Uploads/Editor/OHALDE%20BILGI%20EDINME%20(1).htm' -%>
```